edxval/api.py

```diff
@@ -336,6 +336,24 @@ def create_or_update_video_transcript(
     return video_transcript.url()
 
 
+def delete_video_transcript(video_id, language_code):
+    """
+    Delete transcript for an existing video.
+
+    Arguments:
+        video_id: id of the video with which transcript is associated
+        language_code: language code of a video transcript
+    """
+    try:
+        video_transcript = VideoTranscript.objects.get(video_id=video_id, language_code=language_code)
```

Comment:
What if video id and language code is not provided? should we not return some error so that frontend can let user know ?